core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/ExtractorInfoForwarder.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.source.ExtractorMediaSource;
+import com.novoda.noplayer.listeners.InfoListeners;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+class ExtractorInfoForwarder implements ExtractorMediaSource.EventListener {
```

Comment:
To avoid huge classes we have an `InfoForwarder` for each of the `ExoPlayer` interfaces that we are interested in monitoring.